pkg/operator/ceph/object/spec.go

```diff
@@ -166,21 +168,32 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 	if err != nil {
 		return v1.PodTemplateSpec{}, err
 	}
-	if kmsEnabled {
-		if c.store.Spec.Security.KeyManagementService.IsTokenAuthEnabled() {
-			vaultFileVol, _ := kms.VaultVolumeAndMount(c.store.Spec.Security.KeyManagementService.ConnectionDetails,
-				c.store.Spec.Security.KeyManagementService.TokenSecretName)
-			tmpvolume := v1.Volume{
-				Name: rgwVaultVolumeName,
-				VolumeSource: v1.VolumeSource{
-					EmptyDir: &v1.EmptyDirVolumeSource{},
-				},
-			}
+	s3Enabled, err := c.CheckRGWS3()
```

Comment:
`CheckRGWS3` is a poor name for this function, though I recognize that is a mistake also shared by `CheckRGWKMS`.  Let's not continue with the poor naming here though.

IMO, this is better as `CheckSSES3Enabled`.